src/server/pps/cmds/cmds.go

```diff
@@ -120,16 +120,18 @@ $ pachctl list-job -p foo bar/YYY
 			if err != nil {
 				return err
 			}
-			outputCommits, err := cmdutil.ParseCommits([]string{outputCommit})
-			if err != nil {
-				return err
-			}
-			var outputCommit *pfs.Commit
-			if len(outputCommits) == 1 {
-				outputCommit = outputCommits[0]
+			var _outputCommit *pfs.Commit
```

Comment:
I think this fix is no longer necessary after Matt's fix this morning.

Reply:
Will unfix.